www/content/events.md

```diff
@@ -207,15 +207,16 @@ document.body.addEventListener('htmx:confirm', function(evt) {
 
 ##### Details
 
-{target: target, elt: elt, path: path, verb: verb, triggeringEvent: event, etc: etc, issueRequest: issueRequest}
+{target: target, elt: elt, path: path, verb: verb, triggeringEvent: event, etc: etc, issueRequest: issueRequest, question: question}
```

Comment:
I think this line can probably be removed altogether, seems like the previous line shouldn't have been there at all looking at the other events' docs 🤔 maybe a copied object that was forgotten and left in here at the time this doc was written?